Cargo.toml

```diff
@@ -32,7 +32,7 @@ camino = "1.1.1" # to replace Path and PathBuf
 clap = { version = "4.0.5", features = ["derive"] } # CLI library
 dirs = "4.0.0" # helper functions (e.g. getting the home directory)
 itertools = "0.10.3" # useful iter traits
-kimchi = { git = "https://github.com/o1-labs/proof-systems", rev = "483bd5bd918ebbc1d5e008ce892d7a0023b50ed1" } # ZKP
+kimchi = { git = "https://github.com/o1-labs/proof-systems"} # ZKP
```

Comment:
btw this won't build nicely if we don't specify a version here, as it will always pull the latest which might stop working at some point, maybe point to `5b4ac1437e7912237be88d97b4b4891b22e3e61f`?